platform/linux-generic/odp_pool.c

```diff
@@ -694,6 +694,7 @@ int buffer_alloc_multi(pool_t *pool, odp_buffer_hdr_t *buf_hdr[], int max_num)
 	pool_cache_t *cache;
 	uint32_t cache_num, num_ch, num_deq, burst;
 	odp_buffer_hdr_t *hdr;
+	uint32_t data[burst];
```

Comment:
This is invalid since `burst` is uninitialized here:
```
odp_pool.c: In function ‘buffer_alloc_multi’:
odp_pool.c:697:2: error: ‘burst’ is used uninitialized in this function [-Werror=uninitialized]
  uint32_t data[burst];
  ^
cc1: all warnings being treated as errors
make[1]: *** [odp_pool.lo] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory `/home/travis/build/Linaro/odp/platform/linux-generic'
make: *** [all-recursive] Error 1
```